Combobox.js

```diff
@@ -575,22 +571,50 @@ define([
 			return new ComboPopup({combobox: this});
 		},
 
+		evaluateFilteringConditions: function (inputElement) {
```

Comment:
Can you add a comment explaining what this method does?   From context it seems to be checking whether or not the Combobox should query the store and show the dropdown, but should make that explicit.